drivers/hmis/app/models/hmis/client_alert.rb

```diff
@@ -0,0 +1,10 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::ClientAlert < Hmis::HmisBase
+  belongs_to :created_by, class_name: 'Hmis::User'
+  belongs_to :client, class_name: 'Hmis::Hud::Client'
```

Comment:
suggest adding the other side of the relationship to the client model. That way if a client record is destroyed, the alerts will be destroyed too
```ruby
class Hmis::Hud::Client
  # ...
  has_many :client_alerts, class_name: '::Hmis::ClientAlert', dependent: :destroy, inverse_of: :client
```

Reply:
Thanks, Theron!